pdns/dnsdist.cc

```diff
@@ -144,10 +144,17 @@ struct DelayedPacket
   ComboAddress origDest;
   void operator()()
   {
-    if(origDest.sin4.sin_family == 0)
-      sendto(fd, packet.c_str(), packet.size(), 0, (struct sockaddr*)&destination, destination.getSocklen());
-    else
-      sendfromto(fd, packet.c_str(), packet.size(), 0, origDest, destination);
+    ssize_t res;
+    if(origDest.sin4.sin_family == 0) {
```

Comment:
Well, the main point is to check the return value to be able to issue an informative message a few lines below. The { } are only there because the "if (res == -1)" line looked confusing to me otherwise.